master/static/srv/get_raw_allocation.sql

```diff
@@ -76,23 +76,26 @@ workloads AS (
                             UNION ALL
                             SELECT
                                 'checkpointing' AS kind,
-                                t.id AS trial_id,
+                                trial_id_task_id.trial_id AS trial_id,
                                 checkpoints_v2.report_time AS end_time
                             FROM
                                 checkpoints_v2
                             JOIN
-                                trials AS t on checkpoints_v2.task_id = t.task_id
+                                trial_id_task_id on checkpoints_v2.task_id = trial_id_task_id.task_id
                             UNION ALL
+                            -- TODO does multiple tasks per trial break this?
```

Comment:
Do you think multiple tasks will break this?

I'm not sure if I completely understand this query to be honest.

The ```PARTITION BY trial_id``` makes me think we could also have cases where ```start_times``` are really far apart due to when allocations are scheduled in the unchanged query.

Reply:
i think it is fine, but maybe we can dig into in a call tomorrow

Reply:
Plan is to remove this endpoint anyway since it deprecated for the next release